app/Main.hs

```diff
@@ -0,0 +1,81 @@
+{-# LANGUAGE FlexibleContexts #-}
+
+module Main where
+
+import Options.Applicative
+import Data.Semigroup ((<>))
+
+import BenchShow
+
+verbose :: Parser Bool
+verbose = switch ( long "verbose" <> short 'v' <>
+    help "Provide more details in the report" )
+
+title :: Parser String
+title = strOption (long "title" <> short 't'
+                  <> metavar "Report title"
+                  <> help "Report title, information like plotted field name etc.")
+
+titleField :: Parser TitleAnnotation
+titleField = flag' TitleField
+    ( long "titlefield"
+    <> help "Additional annotation for title of report" )
+
+
+titleEstimator :: Parser TitleAnnotation
+titleEstimator = flag' TitleEstimator
+    ( long "titleestimator"
+    <> help "Additional annotation for title of report" )
+
+titleDiff :: Parser TitleAnnotation
+titleDiff = flag' TitleDiff
+    ( long "titlediff"
+    <> help "Additional annotation for title of report" )
+
+titleAnnotation :: Parser TitleAnnotation
+titleAnnotation = titleField <|> titleEstimator <|> titleDiff
+
+outputDir :: Parser FilePath
+outputDir = strOption
+        (long "outputDir" <> short 'o'
+        <> metavar "Output Directory"
+        <> value "charts"
+        <> help "Directory where output graph or report file should be placed")
+
+groupStyle :: Parser GroupStyle
+groupStyle = flag' Absolute
+    ( long "Absolute" <> help "Show absolute field values for all groups" )
```

Comment:
On CLI usually all lower case is preferred (e.g. `absolute` instead of `Absolute`) for typing convenience.

Reply:
For multi-value config options an argument option instead of using individual flags is a better way to present via CLI, for example `--group-style absolute` or `--group-style=absolute`.